.github/workflows/lint.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 The OpenXLA Authors
```

Comment:
I'm not sure if we need to relicense these files?

Reply:
This isn't relicensing. Just changing the authors line and this is a superset of the "IREE Authors" although I'm not actually sure that we've got it properly defined somewhere. We should probably put an AUTHORS file in the openxla/community repo, but this is fine for now

Reply:
Thanks for confirmation.